### DISCUSSION TITLE
[hive catalog]ipv6 problem

### DISCUSSION THREAD

**Discussion by eyesmoons** [apache/doris#35808] [A - General / Q&A] [answered]
when I create a hive catalog in doris with ipv6 address，but meet an exception when execute the select query ：
select * from hive.sjzt.dmeo limit 1;
ERROR 1105 (HY000): errCode = 2, detailMessage = get file split failed for table: demo, err: java.net.URISyntaxException: Malformed IPv6 address at index 8: hdfs://[xxxx.xxxx.xxxx.xxxx.x.x.x.xx]:8020

**Comment by ixzc** [comment, marked as answer]
at present, IPv6 is not supported.
